### ISSUE TITLE
Travis and git+ssh: github uris

### ISSUE THREAD

**Issue by vivienlacourba** [w3c/respec#972]
@marcoscaceres the WebRTC WG is using respec in its CI and since the change https://github.com/w3c/respec/commit/b8e09b8f97e8794313c77d20a026c1b61aae2859 (using `git+ssh://` github uris) Travis now stops with a `Permission denied` error (see https://travis-ci.org/w3c/webrtc-pc/builds/176385617).
Can you revert or use `https://` github uris instead ?

I am seeing this also broke respec's CI: https://travis-ci.org/w3c/respec/builds/176263945

You can reproduce the issue if you clone one of those repor with a user that don't have an SSH key registered in Github.
```bash
toto@han-solo:/tmp$ git clone git+ssh://git@github.com/isagalaev/highlight.js
Cloning into 'highlight.js'...
...
Warning: Permanently added 'github.com,192.30.253.113' (RSA) to the list of known hosts.
Permission denied (publickey).
```
copying @deniak @dontcallmedom

**Comment by marcoscaceres** [comment]
Yep, sorry about that. Will fix today!

**Comment by vivienlacourba** [comment]
Thx @marcoscaceres I'm also hoping you can release a new version including the switch back to npm (as yarn also brought some additional issues in our CI).

**Comment by marcoscaceres** [comment]
@vivienlacourba, np... can you let me know what the issues are? I would still like to move back to yarn eventually, as it's hella fast during development ... when it works :)

**Comment by marcoscaceres** [comment]
@vivienlacourba, ok, should be good now. 

Be careful relying on the "develop" branch tho... it's deliberately unstable as it's where we do active development.

**Comment by marcoscaceres** [comment]
> Thx @marcoscaceres I'm also hoping you can release a new version including the switch back to npm (as yarn also brought some additional issues in our CI).

Released 6.1.3

**Comment by vivienlacourba** [comment]
Thanks @marcoscaceres !

We usually use the stable `gh-pages` branch, but as we faced issues with yarn on travis I wanted to switch back to code that used npm and after seeing that you had already reverted respec to use npm in your latest commits I decided to give `develop` a shot.
Unfortunately it had the `git+ssh` uri problem reported in this issue.

Regarding the yarn+travis issue we faced it was a ENOENT error on a file in yarn's cache directory, sorry I don't have the travis logs anymore as I earlier re-ran all our failed builds after I reverted to a respec version that used npm.
